remoteClientLib/jni/libs/build-deps.sh

```diff
@@ -492,7 +493,7 @@ build() {
         fi
 
         echo "Copying local recipes into cerbero"
-        git clone https://github.com/iiordanov/remote-desktop-clients-cerbero-recipes.git recipes || true
+        git clone https://github.com/iiordanov/remote-desktop-clients-cerbero-recipes.git -b bugfix/spice-usb-location recipes || true
```

Comment:
Reference here should be put back to master once https://github.com/iiordanov/remote-desktop-clients-cerbero-recipes/pull/1 is merged